File: swagger/models/get_schedule_items_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse
from .staff import Staff

class GetScheduleItemsResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		staff_members
		Contains information about staff members with schedule items.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'StaffMembers': ['staff_members', Staff],
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'staff_members': 'StaffMembers',
		}

	pagination_response = None
	staff_members = None

File: swagger/Models/GetScheduleItemsResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetScheduleItemsResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Contains information about staff members with schedule items.
	 * 
	 * @var Staff[] 
	 */
	public $staffMembers;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'StaffMembers' => ['staffMembers', 'array', Staff::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'staffMembers' => ['StaffMembers', 'array', Staff::class],
			];
	}
}
